src/proxy_auth_v2.lua

```diff
@@ -39,6 +39,29 @@ local err_500_and_log = function(detail, err)
 end
 -- END FUNCTION DEFINITIONS -----–-----–-----–-----–-----–-----–-----–-----–-----–-------
 
+local req = ngx.req
+local req_method = req.get_method()
+local req_uri = ngx.var.request_uri  -- pre-rewrite URI
+local uri = ngx.var.uri  -- post-rewrite URI
+
+-- BEGIN OPEN ENDPOINT LOGIC ------------------------------------------------------------
+
+-- Pass through all endpoint calls which used to be proxied by bento_public
+-- TODO: replace this with properly authorization-compatible services
+
+if req_method == "GET" and (
+  uri == "/service-info" or
+  req_uri == "/api/metadata/api/projects" or
+  req_uri == "/api/metadata/api/public" or
```

Comment:
I think we should strip URL params from `req_uri` before we check for passthrough.
Public uses URL params for search on this endpoint (e.g. `/api/metadata/api/public?sex=MALE`), passthrough only works if there are no params.

Reply:
good point

Reply:
Works now, thanks!